core/templates/pages/terms-page/terms-page.mainpage.html

```diff
@@ -318,6 +317,9 @@ <h2>Changelog</h2>
           list-style-type: none;
           padding-left: 20px;
         }
+        .oppia-refrain-from-activity-header {
```

Comment:
> That said, this seems like an extremely special case -- styling should be applied uniformly. If you get rid of the additional style, does the page look bad? If not then getting rid of it is fine, but please check to ensure that there are no unintended visual side effects.

You have not answered most of my questions here. I'm wondering whether the class name is even necessary.

Please make sure to thoroughly address review comments, otherwise it's as if they haven't been replied to.